MaskRCNN_no_batch_convergence/model_fpn.py

```diff
@@ -216,8 +280,8 @@ def multilevel_rpn_losses(
                 name_scope='level{}'.format(lvl + 2))
             losses.extend([label_loss, box_loss])
 
-        total_label_loss = tf.add_n(losses[::2], name='label_loss')
-        total_box_loss = tf.add_n(losses[1::2], name='box_loss')
+        total_label_loss = tf.add_n(losses[::2], name='label_loss' if name else None)
```

Comment:
Can probably get rid of these two lines now and have it back to the way it used to be.  Originally, It was to avoid the tensor naming issue for tensorboard.  It looks like the name_scope is being intentionally renamed within `multilevel_rpn_losses_batch_fixed_single_image` so it doesn't seem like it would appear to be a problem anymore.

Reply:
Fixed